riscv/sim.h

```diff
@@ -71,13 +70,12 @@ class sim_t : public htif_t, public simif_t
 
 private:
   isa_parser_t isa;
+  const cfg_t *cfg;
```

Comment:
Yep, can do. I don't think it really matters (because the only thing that could happen with this code is that we point `cfg` at some other const configuration), but happy to make the change.